src/block.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 Object.defineProperty(exports, '__esModule', { value: true });
 exports.Block = void 0;
+const _1 = require('.');
```

Comment:
More of a nit than anything: Is there any reason these couldn't be rewritten to something like:

```ts
const networks = require('./networks');
```

Or is there a reason for the _1 naming I'm missing?

Reply:
The .js files are generated from .ts. So the JavaScript files are not manually edited. They are included for security and review reasons of library consumers.

Reply:
Thanks for calling that out - I should have realized I was reading the js instead of ts file.